vunit/modelsim_interface.py

```diff
@@ -276,7 +276,8 @@ def _create_load_function(self,  # pylint: disable=too-many-arguments,too-many-l
         else:
             coverage_file = join(output_path, "coverage.ucdb")
             self._coverage_files.add(coverage_file)
-            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" % fix_path(coverage_file)
+            coverage_save_cmd = "coverage save -onexit -assert -directive -cvg -codeAll {%s}" \
+			% (os.path.basename(os.path.dirname(output_path)), fix_path(coverage_file))
```

Comment:
I am missing a comment or intermediate variable name to document the intention that dirname of output path is the test name.